betelgeuse/parser.py

```diff
@@ -187,3 +187,31 @@ def parse_docstring(docstring=None):
             field_value = output
             fields_dict[field_name] = field_value
     return fields_dict
+
+
+def parse_markers(all_markers=None, config=None):
+    """Parse the markers."""
+    resolved_markers = []
+    ignore_list = getattr(config, 'MARKERS_IGNORE_LIST', None)
+
+    def _process_marker(marker):
+        # Fetching exact marker name
+        marker_name = marker.split('mark.')[-1] if 'mark' in marker else marker
```

Comment:
```suggestion
        marker_name = marker.split('mark.')[-1]
```

I don't think that `if 'mark' in marker else marker` is necessary here:
```
In [8]: marker.split('mark.')[-1]
Out[8]: 'fero.jozo.lojzo'

In [9]: marker = 'mark.fero.jozo.lojzo'

In [10]: marker.split('mark.')[-1]
Out[10]: 'fero.jozo.lojzo'

In [11]: marker = 'fero.jozo.lojzo.mark'

In [12]: marker.split('mark.')[-1]
Out[12]: 'fero.jozo.lojzo.mark'
```